network-api/networkapi/templates/fragments/breadcrumbs.html

```diff
@@ -1,8 +1,8 @@
 
-<div id="breadcrumb-container">
+<div id="breadcrumb-container" class="tw-pr-8">
```

Comment:
Adding this padding to the right of the container because I found on mobile screens the breadcrumbs and trailing "/" characters were getting very close to the "cutout".


**Before:**

<img width="400" src="https://github.com/MozillaFoundation/foundation.mozilla.org/assets/18314510/0eedda8f-b5e1-42fa-a697-035dd0f9c1ad">


**After:**

<img width="400" src="https://github.com/MozillaFoundation/foundation.mozilla.org/assets/18314510/9141a4a9-9eca-4163-a366-a326b939d036">